nssp_demo/postprocess_state_forecast.R

```diff
@@ -251,9 +247,50 @@ make_forecast_figs <- function(model_dir,
     )
   )
 
-  all_forecast_plots
+  return(all_forecast_plots)
 }
 
+
+theme_set(theme_minimal_grid())
+
+disease_name_formatter <- c("covid-19" = "COVID-19", "influenza" = "Flu")
+disease_name_nssp_map <- c(
+  "covid-19" = "COVID-19/Omicron",
+  "influenza" = "Influenza"
+)
+
+# Create a parser
+p <- arg_parser("Generate forecast figures") |>
+  add_argument(
+    "--model-dir",
+    help = "Directory containing the model data",
+  ) |>
+  add_argument(
+    "--filter-bad-chains",
+    help = "Filter out bad chains from the samples",
+    flag = TRUE
+  ) |>
+  add_argument(
+    "--good-chain-tol",
+    help = "Tolerance level for determining good chains",
+    default = 2L
+  )
+
+argv <- parse_args(p)
+model_dir <- path(argv$model_dir)
+filter_bad_chains <- argv$filter_bad_chains
+good_chain_tol <- argv$good_chain_tol
```

Comment:
another opportunity for [`list2env`](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/list2env)